packages/back-end/src/models/OrganizationModel.ts

```diff
@@ -33,11 +33,19 @@ const organizationSchema = new mongoose.Schema({
     },
   ],
   stripeCustomerId: String,
+  discountCode: String,
+  freeSeats: Number,
   subscription: {
     id: String,
     qty: Number,
     trialEnd: Date,
     status: String,
+    current_period_end: Number,
+    cancel_at: Number,
+    canceled_at: Number,
+    cancel_at_period_end: Boolean,
+    planNickname: String,
+    priceId: String,
```

Comment:
Having this nested under `subscription` is going to make it hard to use for grandfathering.  Throughout the code, we're checking if `organization.subscription` exists and if so, assuming the person has an active subscription.  So there's no way to give them a different priceId when they don't have a subscription already.

Reply:
@jdorn Let's talk through this on Monday - I'm not sure I follow. The mental model I put together for how to support grandfathering organizations all use a single priceId (the difference would be the number of `freeSeats` and any `discountCode`.